code/datums/elements/soft_landing.dm

```diff
@@ -17,9 +17,17 @@
 	UnregisterSignal(target, COMSIG_ATOM_INTERCEPT_Z_FALL)
 
 ///signal called by the stat of the target changing
-/datum/element/soft_landing/proc/intercept_z_fall(obj/soft_object, falling_movables, levels)
+/datum/element/soft_landing/proc/intercept_z_fall(atom/soft_object, falling_movables, levels)
 	SIGNAL_HANDLER
 
-	for (var/mob/living/falling_victim in falling_movables)
-		to_chat(falling_victim, span_notice("[soft_object] provides a soft landing for you!"))
+	for(var/mob/living/falling_victim in falling_movables)
+		var/turf/falling_spot = get_turf(falling_victim)
+		
+		if(locate(/obj/structure/stairs) in falling_spot)
+			return FALL_INTERCEPTED | FALL_NO_MESSAGE
```

Comment:
why are you running this on the loop

Reply:
https://github.com/tgstation/tgstation/issues/68466#issuecomment-1186614037

Whenever slime people use the stairs they get a soft landing message.

Reply:
https://github.com/tgstation/tgstation/pull/69263/commits/8cc049775012a53c72b89f63e543355c7cb8326e